addons/web_editor/static/src/js/wysiwyg/widgets/media_dialog.js

```diff
@@ -159,6 +159,14 @@ var MediaDialog = Dialog.extend({
      * @override
      */
     save: function () {
+        if (this.activeWidget.error) {
+            this.displayNotification({
+                type: 'danger',
+                title: _t("You can't add this media"),
```

Comment:
why not using the error we already got? (If I read this correctly)
```suggestion
                title: this.activeWidget.error,
```

Reply:
Notifications need both a title and a massage. Here I use the existing error string for the message but I add one for the title

Reply:
No, notification don't need both (fortunately), here just omit it, it's better than introducing an useless term (the message is already clear enough) that will never be translated.
Bonus, it makes the diff minimal :+1: 
In master you can add it if you prefer (I wouldn't but it's fine either ways).